forem/forem
# refactor: Use keyboard shortcut hook in article form

## What type of PR is this? (check all applicable)


## Description
Now we have a helper for keyboard shortcuts we can move old shortcuts over to this new method

## Related Tickets & Documents
#10878

## QA Instructions, Screenshots, Recordings
Press ctrl/command + shift + p in the new post page
it should toggle the preview

## Added tests?


## Added to documentation?


## [optional] Are there any post deployment tasks we need to perform?
N/A

## [optional] What gif best describes this PR or how it makes you feel?

![Mad laughter](https://media.giphy.com/media/3oFzmnZSb3YNuCbUiY/giphy.gif)
